build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -0,0 +1,180 @@
+
+      <Copy SourceFiles="$(BinaryToCorehostifyOutDir)/csc.exe"
+            DestinationFiles="$(SdkOutputDirectory)/csc.dll" />
+
+      <Move SourceFiles="$(BinaryToCorehostifyOutDir)/csc.exe"
+            DestinationFiles="$(SdkOutputDirectory)/csc.exe" />
+
+      <Copy SourceFiles="$(SdkOutputDirectory)/compilers.deps.json"
+            DestinationFiles="$(SdkOutputDirectory)/csc.deps.json" />
+
+      <Copy SourceFiles="$(SdkOutputDirectory)/compilers.runtimeconfig.json"
+            DestinationFiles="$(SdkOutputDirectory)/csc.runtimeconfig.json" />
+
+      <ChangeEntryPointLibraryName 
+            DepsFile="$(SdkOutputDirectory)/csc.deps.json"
+            NewName="csc.deps.json" />
+
+      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/csc.deps.json"
+                                   SectionName="runtimeTargets"
+                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
+      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/dotnet.deps.json"
+                                   SectionName="runtimeTargets"
+                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
+
+      <!-- cleanup compilers project output we don't need -->
+      <CleanPublishOutput Path="$(SdkOutputDirectory)"
+                          Name="compilers"
+                          DeleteRuntimeConfigJson="true"
+                          DeleteDepsJson="true" />
+
+      <!-- Copy Host to SDK Directory -->
+      <Copy SourceFiles="$(SharedFrameworkNameVersionPath)/$(DotnetHostBaseName)"
+            DestinationFiles="$(SdkOutputDirectory)/corehost$(ExeSuffix)" />
+
+      <Copy SourceFiles="$(SharedFrameworkNameVersionPath)/$(DotnetHostFxrBaseName)"
+            DestinationFiles="$(SdkOutputDirectory)/$(DotnetHostFxrBaseName)" />
+
+      <Copy SourceFiles="$(SharedFrameworkNameVersionPath)/$(HostPolicyBaseName)"
+            DestinationFiles="$(SdkOutputDirectory)/$(HostPolicyBaseName)" />
+
+      <CrossgenDirectory SharedFrameworkNameVersionPath="$(SharedFrameworkNameVersionPath)"
+                         SdkOutputDirectory="$(SdkOutputDirectory)" />
+
+      <!-- Generate .version file -->
+      <WriteLinesToFile File="$(SdkOutputDirectory)/.version"
+                        Lines="$(CommitHash)$(NewLine)$(NugetVersion)$(NewLine)"
+                        Overwrite="true" />
+
+      <Copy SourceFiles="@(MSBuildTargetsToCopy)"
+            DestinationFiles="@(MSBuildTargetsToCopy->'$(SdkOutputDirectory)\%(RecursiveDir)%(Filename)%(Extension)')" />
+
+      <ItemGroup>
+        <FilesToClean Include="$(StageDirectory)/sdk/**/vbc.exe" />
+      </ItemGroup>
+
+      <Delete Files="@(FilesToClean)" />
+
+      <ItemGroup>
+        <FilesToCopy Include="$(StageDirectory)/**/*.*" />
+        <PdbsToClean Include="$(StageDirectory)/sdk/**/*.pdb" />
+      </ItemGroup>
+
+      <Copy SourceFiles="@(FilesToCopy)"
```

Comment:
That's not what we do today. We copy everything. The difference is that then we will remove the pbds from the actual stage. So, we end up with two full stages. We can change that if we want, but not how it is being done currently.